docs/Compose.md

```diff
@@ -0,0 +1,181 @@
+# Running GUAC with Docker Compose
+
+This tutorial introduces the full GUAC ingestion pipeline and how to deploy it
+with the included Docker Compose configuration.
+
+## Demo GUAC Ingestion
+
+The GUAC demonstrations use the full GUAC GraphQL server, but use a demo-level
+all-in-one ingestion command `guacone files`. This command runs the GUAC
+collector, parser, assembler, and ingestor components all at once. These
+components are designed to be run as separate asynchronous services that combine
+to form a robust and scalable ingestion pipeline.
+
+## GUAC Components
+
+### Server
+
+The GraphQL server serves the GUAC defined nodes through GraphQL queries. It is
+an abstraction layer for GUAC integrations and other GUAC components. This
+tutorial uses the built-in in-memory backend for the server. Currently the
+server also supports a Neo4j backend. Any future backend database support added
+to GUAC will not affect the GraphQL interface that the server provides.
+
+### Ingestion
+
+| Name      | Short Description                                                                 |
+| --------- | --------------------------------------------------------------------------------- |
+| Ingestor  | Takes completed GUAC objects and performs GraphQL mutation queries to add to GUAC |
+| Assembler | Takes parsed document objects and assembles GUAC objects                          |
+| Parser    | Takes documents (ex: SBOMs) and parses the objects inside them                    |
+| Collector | Reads or watches locations for new documents and collects them when found         |
```

Comment:
may be helpful to rearrange in order that they run, collect, parse, ingest, assemble .. (although i think we threat what's described here as Ingestor and Assembelr as Assembler, and the parser is the ingestor.